src/hooks/useAuthorizeAgent.ts

```
@@ -9,7 +8,7 @@ import { AGENT_CONTRACT } from '@/utils/monarch-agent';
 import { MONARCH_TX_IDENTIFIER, MORPHO } from '@/utils/morpho';
 import { SupportedNetworks } from '@/utils/networks';
 import { Market } from '@/utils/types';
-
+import { useStyledToast } from '@/hooks/useStyledToast';
```

ESLint

[error] `@/hooks/useStyledToast` import should occur before import of `@/hooks/useTransactionWithToast`

(import/order)